clustering/server/src/main/java/org/wildfly/clustering/server/singleton/DistributedSingletonServiceConfigurator.java

```diff
@@ -1,5 +1,4 @@
 /*
- * JBoss, Home of Professional Open Source.
```

Comment:
Not a big deal probably, but is there a reason this was deleted?

Reply:
I think that may have been the cat walking across my keyboard while I was making coffee. I'll fix.